ts/components/wallet/payWith/addNewPaymentMethodComponent.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * This component will display the payment method that can be registered
+ * on the app
+ * TODO: replace the contextual help with the appropriate
+ *  component @https://www.pivotaltracker.com/story/show/157874540
+ */
+
+import color from "color";
+import { Left, ListItem, Right, Text, View } from "native-base";
+import * as React from "react";
+import { FlatList, StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../../i18n";
+import ROUTES from "../../../navigation/routes";
+import Icon from "../../../theme/font-icons/io-icon-font/index";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  isInTransaction?: boolean;
```

Comment:
it looks like this property is never read, also why it's optional? it's either inside a transaction or it's not

Reply:
@valentinacala remove the question mark